README.md

```diff
@@ -113,7 +113,7 @@ These icons are released within in the bundle package as `@spectrum-css/spectrum
 loadIcons('node_modules/@adobe/spectrum-css-workflow-icons/dist/spectrum-icons.svg');
 ```
 
-You can then use the icons in your app. Visit the [Spectrum CSS workflow icon list](http://opensource.adobe.com/spectrum-css/icons/) and click on any icon to get the markup.
+You can then use the icons in your app. Visit the [Spectrum CSS workflow icon list](https://spectrum.adobe.com/page/icons/) and click on any icon to get the markup.
```

Comment:
Let's change the link text to be `Spectrum workflow icon list`

Reply:
Let me arrange that!